.github/workflows/self-host-install.yml

```diff
@@ -15,27 +15,26 @@ on:
 jobs:
   touca-self-host-install:
     name: touca-self-host-install
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
Limiting this to Linux for now until we find time to investigate issue with installing on Windows in CI.